.github/workflows/publish.yml

```diff
@@ -19,16 +19,18 @@ jobs:
           java-version: 1.8
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
+      - name: Prepare
```

Comment:
what do we prepare? it would be good to give more details to make it easier to understand for someone who is not acquainted with the setup. for example, the previous step is very clear: "Grant execute permission for gradlew"